src/ui/pages/Settings/Settings.tsx

```diff
@@ -303,7 +303,23 @@ function DeveloperTools() {
                 testnetMode: event.target.checked ? { on: true } : null,
               });
             }}
-            detailText="Enables viewing and interacting with test networks"
+            detailText={
+              <div>
+                Enables viewing and interacting with test networks <br />
```

Comment:
```suggestion
                Enables viewing and interacting with test networks. <br />
```
Not sure about adding the period here, looks like it was omitted intentionally